src/app/shared-component/dialog/dialog.component.ts

```diff
@@ -12,6 +12,28 @@ import { Dialog } from './dialog';
   selector: 'pf-dialog',
   templateUrl: './dialog.component.html',
   styleUrls: ['./dialog.component.scss'],
+   animations: [
+    trigger('modalState', [
+      state('inactive', style({
+        opacity: '0',
+        transform: 'translateY(-50%)'
+      })),
+      state('active', style({
+        opacity: '1',
+        transform: 'translateY(0)'
+      })),
+      transition('inactive <=>active', animate('500ms ease all'))
+    ]),
+    trigger('modalOverlay', [
+      state('inactive', style({
+        background: 'transparent'
+      })),
+      state('active', style({
+        background: 'rgba(0,0,0,0.4)'
```

Comment:
@SMahil just a consistency bit, IMO, but not a problem :)

@sanbornsen you can set those with opacity effects and layering, so makes no difference